lib/perl/Genome/Qc/Command/BuildMetrics.t

```diff
@@ -49,7 +55,7 @@ my $pkg = 'Genome::Qc::Command::BuildMetrics';
 use_ok($pkg);
 
 my $test_dir = __FILE__.'.d';
-my $expected_output_file = $test_dir .'/expected_build_metrics.yml';
+my $expected_output_file = $test_dir .'/expected_build_metrics_v2.yml';
```

Comment:
Since this is in the repo, we can edit the original `expected_build_metrics.yml` if nothing else is using it.